i18n/locales/de/common.json

```diff
@@ -1,238 +1,532 @@
+  "Your signed message": "Deine signierte Nachricht",
+  "Your voice matters": "Deine Stimme zählt.",
+  "You’ll see it in Delegates and it will be confirmed in a matter of minutes.": "You’ll see it in Delegates and it will be confirmed in a matter of minutes.",
+  "from": "von",
+  "or": "oder",
+  "{{deviceModel}} connected! Open the Lisk app on the device": "{{deviceModel}} verbunden! Öffne die Lisk App auf dem Gerät",
+  "{{fee}} {{token}}": "{{fee}} {{token}}",
+  "{{length}} bytes left": "{{length}} verbleibende Bytes",
+  "{{length}} extra characters": "{{length}} verbleibende Zeichen",
+  "{{length}} out of {{maxLength}} characters left": "{{length}} von {{maxLength}} verleibende Zeichen ",
+  "{{passphraseName}} paper wallet": "{{passphraseName}} Papier wallet",
+  "{{token}} Balance": "{{token}} Kontostand",
+  "{{token}} Wallet": "{{token}} Wallet",
+  "{{token}} balance": "{{token}} Kontostand",
+  "Account Balance:          {{balance}} {{token}}": "Account Balance:          {{balance}} {{token}}",
+  "Address already bookmarked": "Address already bookmarked",
+  "Become a delegate summary": "Become a delegate summary",
+  "Bookmarked": "Bookmarked",
+  "DD MMM YY, HH:mm": "DD MMM YY, HH:mm",
+  "DD MMM YYYY": "DD MMM YYYY",
+  "DD MMM YYYY, hh:mm A": "DD MMM YYYY, hh:mm A",
+  "DD.MM.YY": "DD.MM.YY",
+  "Discord": "Discord",
+  "Discreet Mode Disabled": "Der Diskreter Modus ist deaktiviert",
+  "Discreet Mode Enabled": "Der Diskreter Modus ist aktiviert",
+  "Each LSK token is worth one vote.": "Each LSK token is worth one vote.",
+  "English": "Englicsh",
+  "Filter by name...": "Filtern nach Namen...",
+  "German": "Deutsch",
+  "Go to Dashboard": "Gehe zum Dashboard",
+  "Go to Wallet": "Gehe zum Wallet",
+  "Go to confirmation": "Gehe zum Bestätigung",
+  "Go to settings": "Gehe zum Einstellungen",
+  "Invalid address": "Ungültige Adresse",
+  "Invalid dates": "Ungültiges Datum",
+  "MMM DD YYYY": "MMM DD YYYY",
+  "MMM DD YYYY hh:mm": "MMM DD YYYY hh:mm",
+  "MMM DD YYYY hh:mm:ss": "MMM DD YYYY hh:mm:ss",
+  "MMM DD YYYY hh[h]": "MMM DD YYYY hh[h]",
+  "MMM YYYY": "MMM YYYY",
+  "Passphrase:": "Passphrase",
+  "Removed votes": "Entfernte Stimmen",
+  "YYYY": "YYYY",
+  "You don’t have enough balance to enable it. {{fee}} LSK is required.": "You don’t have enough balance to enable it. {{fee}} LSK is required.",
+  "Your {{walletModel}} has an old Firmware. Please update it.": "Your {{walletModel}} has an old Firmware. Please update it.",
+  "[Today], hh:mm A": "[Today], hh:mm A",
+  "[Tomorrow], hh:mm A": "[Tomorrow], hh:mm A",
+  "[Yesterday], hh:mm A": "[Yesterday], hh:mm A",
+  "ie. 192.168.0.1": "ie. 192.168.0.1",
+  "ie. Lisker123": "ie. Lisker123",
+  "ie. peter_pan": "ie. peter_pan",
+  "just now": "gerade jetzt",
+  "secondPassphraseInfoParagraph": "secondPassphraseInfoParagraph",
```

Comment:
This translation doesn't correspond to the English version in https://github.com/LiskHQ/lisk-hub/blob/f512ee1d458a95dbf9d2c6898a29740ec93e29a4/i18n/locales/en/common.json#L506

Reply:
Apparently used as key

Reply:
Not sure what you mean with "Apparently used as key".
The problem is that the german version shows "secondPassphraseInfoParagraph" instead of a translation.
I if the translation is not yet available, the best practice is AFAIK to show the English (by deleting `secondPassphraseInfoParagraph` key from `i18n/locales/de/common.json` )

<img width="778" alt="Screenshot 2019-09-05 at 13 19 25" src="https://user-images.githubusercontent.com/1254342/64337571-f1a09380-cfdf-11e9-8c5c-a6cfff3627b2.png">
<img width="763" alt="Screenshot 2019-09-05 at 13 19 56" src="https://user-images.githubusercontent.com/1254342/64337573-f2392a00-cfdf-11e9-837f-ee906a7c747a.png">